packages/hdwallet-native/src/binance.ts

```diff
@@ -1,13 +1,16 @@
 import * as core from "@shapeshiftoss/hdwallet-core";
 import * as bech32 from "bech32";
 import BigNumber from "bignumber.js";
-import type * as bnbSdk from "bnb-javascript-sdk-nobroadcast";
+import type * as bnbSdkTypes from "bnb-javascript-sdk-nobroadcast";
 import CryptoJS from "crypto-js";
+import PLazy from "p-lazy";
 
 import * as Isolation from "./crypto/isolation";
 import { NativeHDWalletBase } from "./native";
 import * as util from "./util";
 
+const bnbSdk = PLazy.from(async () => await import("bnb-javascript-sdk-nobroadcast"));
```

Comment:
I don't understand this. What's wrong with:
```suggestion
const bnbSdk = PLazy.from(() => import("bnb-javascript-sdk-nobroadcast"));
```

Reply:
nothing in particular, I just have a general aversion to returning promises without awaiting them; makes for better stack traces imo.

Reply:
applied your suggestion on all similar lines.